### PR TITLE
Fix explainer in multiple paid-for MPUs

### PR DESCRIPTION
**Pull request by janua**
When there is more than one `paidFor` MPU on the page, the about button breaks.

This fixes `getTarget` used to get the toggle target so that is searches from the parent only instead of the whole document.

@regiskuckaertz

### CODE REVIEW

**Comment by janua** [comment]
[Multiple paid content MPUs broken about behaviour](https://trello.com/c/G5PtdpqY/90-multiple-paid-content-mpus-broken-about-behaviour)

File: static/src/javascripts/projects/common/modules/ui/toggles.js [hunk 1]
```diff
@@ -63,10 +63,10 @@ define([
         });
     };
 
-    Toggles.prototype.getTarget = function (control) {
+    Toggles.prototype.getTarget = function (parent, control) {
         var targetClass = bonzo(control).data('toggle');
         if (targetClass) {
-            return document.body.querySelector('.' + targetClass);
+            return parent.querySelector('.' + targetClass);
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/ui/toggles.js]: LOL, indeed.

File: static/src/javascripts/projects/common/modules/ui/toggles.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ define([
 
             controls.forEach(function (control) {
                 if (!bonzo(control).hasClass(readyClass)) {
-                    var target = self.getTarget(control);
+                    var target = self.getTarget(parent, control);
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/ui/toggles.js]: Is it `parent` or `component`?

File: static/src/javascripts/projects/common/modules/ui/toggles.js (hunk 1 above)
Comment (regiskuckaertz) [inline static/src/javascripts/projects/common/modules/ui/toggles.js]: (line 22)

**Review by regiskuckaertz — CHANGES_REQUESTED** [review]

**Comment by janua** [comment]
@regiskuckaertz Now using `component` in `getTarget` call.

**Review by regiskuckaertz — APPROVED** [review]

**Comment by regiskuckaertz** [comment]
Yiihaaa

**Pull request merged**